src/Core/Security/Authentication/Domain/Provider/OpenIdProvider.php

```diff
@@ -1235,4 +1239,122 @@ public function getRolesMappingFromProvider(): array
     {
         return $this->rolesMappingFromProvider;
     }
+
+    /**
+     * @param GroupsMapping $groupsMapping
+     */
+    private function validateGroupsMappingOrFail(GroupsMapping $groupsMapping): void {
+        if ($groupsMapping->isEnabled()) {
+            $groups = $this->getGroupsFromProvider($groupsMapping->getEndpoint());
+            $this->validateGroupsMapping($groups, $groupsMapping);
+        } else {
+            $this->logAuthenticationInfo("Groups Mapping disabled", []);
```

Comment:
message will be formatted like `"[Openid] [INFO] $message : " . json_encode($content)`
so `"[Openid] [INFO] Groups Mapping disabled : {}"`
maybe, logAuthenticationInfo should be updated to not log content when it's empty